client/my-sites/hosting/main.js

```diff
@@ -5,7 +5,87 @@
  */
 
 import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+import { localize } from 'i18n-calypso';
 
-const Hosting = () => <p>Placeholder</p>;
+/**
+ * Internal dependencies
+ */
+
+import Main from 'components/main';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import DocumentHead from 'components/data/document-head';
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import Button from 'components/button';
+import MaterialIcon from 'components/material-icon';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const DataSection = ( { title, data } ) => {
+	return (
+		<p>
+			<strong>{ title }: </strong>
+			{ data }
+		</p>
+	);
+};
+
+const Hosting = ( { translate } ) => {
+	const dummyInfo = {
+		[ translate( 'URL' ) ]: 'sftp1.wordpress.com',
+		[ translate( 'Port' ) ]: 22,
+		[ translate( 'Username' ) ]: 'test',
+		[ translate( 'Password' ) ]: 'test',
+	};
+
+	return (
+		<Main className="hosting is-wide-layout">
+			<PageViewTracker path="hosting/:site" title="Hosting" />
```

Comment:
We can verify analytics calls in console by calling:

```
localstorage.debug = 'calypso:analytics:*';
```
and refreshing the page.

<img width="978" alt="Screen Shot 2019-10-02 at 12 39 36 PM" src="https://user-images.githubusercontent.com/1270189/66076052-cabb7b80-e511-11e9-9bf5-e2ef7b7509c6.png">

Do you mind double-checking if we have a site type of `Atomic` or if we just bucket everyone else as `Jetpack`? This is likely unrelated, but we might have some page stats tracking issue we could file an issue for / fix.

Reply:
Looks like we're just bucketing everything into `jetpack`, it seems. Wasn't able to get another site type to show up, then found this:

<img width="861" alt="Screen Shot 2019-10-06 at 11 37 24 PM" src="https://user-images.githubusercontent.com/8892849/66283674-5dea0d80-e892-11e9-80d4-b3279516598f.png">